test/runtests.jl

```diff
@@ -68,6 +68,120 @@ end
                    8.848886704358202, 2.8697881579099085, 11.15107375630744, 11.8392578374927])
 end
 
+@testset "rankdeficient-glm" begin
+    Random.seed!(9999)
+    df = DataFrames.DataFrame()
+    num_rows = 100
+    df[:feature_1] = randn(num_rows)
+    df[:feature_2] = 2 .* df[:feature_1]
```

Comment:
No need for the broadcast, `2 * df[:feature_1]` does the same.

Reply:
Done.